### ISSUE TITLE
Test Level: If you die in test level, the next time you respawn, any blood on the HUD is not cleared

### ISSUE THREAD

**Issue by rvanner** [TheGameCreators/GameGuruRepo#2340]
It's not so easy to see but it is there in this shot (I have seen it worse):

![image](https://user-images.githubusercontent.com/1503593/156126201-f30ecb96-babe-4040-b572-1a5354315ebd.png)

**Comment by LeeBamberTGC** [comment]
@dllllllllllb Nice one for you to trace and resolve.

**Comment by dllllllllllb** [comment]
Just to justify why it took me almost 3 hours to change 2 values.
At first, I thought there is something wrong with the way blood splats are reset, so I played around with values, I tried to completely reset them to initial values from their constructor. I also tried moving them off the screen when unloading level. Then I thought that there might be a rouge blood splat that is being spawned somewhere, tried looking for that. After that, I tried to reset the entire HUD using ResetSpriteBatcher and hud_free.

Then I noticed that the splats are always in the top left corner, so I thought it might be something with the x/y constructor values, tried changing them to be off screen but that didn't work either. Then I had the idea that there might be something wrong with the sprites, so I wanted to find where they are loaded in the code and found where the bug is by accident.
